packages/fxa-payments-server/src/components/PaymentFormV2/index.test.tsx

```diff
@@ -0,0 +1,516 @@
+import React from 'react';
+import TestRenderer from 'react-test-renderer';
+import { render, cleanup, act, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import waitForExpect from 'wait-for-expect';
+
+import { Omit } from '../../lib/types';
```

Comment:
Huh!  Could use TS's built in Omit instead?

Reply:
This is copied over from the existing PaymentForm tests. I think at the time this was written, TS didn't have Omit yet. Can try dropping our definition of it though